app/services/iiif_manifest.rb

```diff
@@ -67,12 +76,19 @@ def add_metadata
     sidecar.update(data: sidecar.data.merge(manifest_metadata))
   end
 
+  def store_timestamps(metadata)
```

Comment:
Is there a better method name here? when invoked as `metadata = store_timestamps` it reads as though it would be adding the timestamps to the metadata, but in fact it removes them.

Reply:
I'm not sure what to name this. metadata = metadata_without_timestamps? Really, I think the only way to make it clearer is to refactor this class to use a value object instead of a hash for the metadata. What do you think?

Reply:
@hackmastera I did a little procedural refactor, maybe this feels better?

Reply:
yes thank you!!